### PR TITLE
add  site association for ios universal linking

### PR DESCRIPTION
**Pull request by sabeurthabti**
associated with https://github.com/quran/quran-ios/issues/195

1. promoting the App
2. Linking between our apps (quranicaudio is coming soon(TM)

### CODE REVIEW

**Comment by sabeurthabti** [comment]
@mohamede1945 to provide me with the appID

**Comment by sabeurthabti** [comment]
should we support staging.quran.com subdomain ? might be the quickest way to test this.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32927

File: src/server/config/express.js [hunk 1]
```diff
@@ -55,6 +56,14 @@ export default (server) => {
     proxyOneQuran.web(req, res);
   });
 
+  server.use('/apple-app-site-association', (req, res) => {
```
Comment (naveed-ahmad) [inline src/server/config/express.js]: Looks like a static file(same as manifest.json), why we need route for this? 

https://quran.com/manifest.json

Comment (sabeurthabti) [inline src/server/config/express.js]: apple dictates it cannot be a `.json` file and need to be served with `application/pkcs7-mime'` content-type

Comment (sabeurthabti) [inline src/server/config/express.js]: can we set the content-type from the static folder? the file cannot have an extension of `.json`

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32928

File: src/server/config/express.js (hunk 1 above)
Comment (mmahalwy) [inline src/server/config/express.js]: I want to say you can use express' static file helper with the route and add content type. Not certain but this works

File: src/server/config/apple-app-site-association.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "webcredentials": {
+    "apps": [
+      "abc.com.quran"
+    ]
+  },
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "abc.com.quran",
```
Comment (mmahalwy) [inline src/server/config/apple-app-site-association.json]: ? that's actually it? `abc`?

**Comment by mohamede1945** [comment]
You can use `MKP73M5X4A.com.quran.ios` as `appId`.

File: src/server/config/apple-app-site-association.json (hunk 2 above)
Comment (sabeurthabti) [inline src/server/config/apple-app-site-association.json]: this is a place holder text, I was waiting for @mohamede1945

**Comment by sabeurthabti** [comment]
@mmahalwy let's merge this in this way to ensure it works, then I am happy to give this another try with static folder approach. 

@mohamede1945  can we support, https://quran.com/ and https://staging.quran.com/

**Comment by sabeurthabti** [comment]
rebuild

**Comment by sabeurthabti** [comment]
rebuild

**Comment by sabeurthabti** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32930

**Pull request merged**